test/Problems/Linear/linear_problems.jl

```diff
@@ -0,0 +1,382 @@
+function lp1_test()
+    #=
+        min -4x_2 - 1
+    st
+        x_1 + 2x_2 <= 3
+        x_1 >= 1
+        x_1 >= 3
+    =#
+
+    model = TestModel{Float64}()
+    
+    X = MOI.add_variables(model, 2)
+    
+    MOI.add_constraint(model, 
+        MOI.ScalarAffineFunction(
+            MOI.ScalarAffineTerm.([1.0, 2.0], X), 0.0),
+            MOI.LessThan(3.0))
+    
+    MOI.add_constraint(model, 
+        MOI.SingleVariable(X[1]),
+            MOI.GreaterThan(1.0))
+    
+    MOI.add_constraint(model, 
+        MOI.SingleVariable(X[1]),
+            MOI.GreaterThan(3.0))
+    
+    MOI.set(model, 
+        MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), 
+        MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([-4.0], [X[2]]), -1.0))
+    
+    MOI.set(model, MOI.ObjectiveSense(), MOI.MIN_SENSE)
+    
+    return model
+end
+
+function lp2_test()    
+    #=     
+        min -4x1 -3x2 -1
+    s.t.
+        2x1 + x2 + 1 <= 4
+        x1 + 2x2 + 1 <= 4
+        x1 >= 1
+        x2 >= 0
+    =#
+    model = TestModel{Float64}()
+    
+    X = MOI.add_variables(model, 2)
+    
+    MOI.add_constraint(model, 
+        MOI.ScalarAffineFunction(
+            MOI.ScalarAffineTerm.([2.0, 1.0], X), 0.0),
+             MOI.LessThan(3.0))
+    
+    MOI.add_constraint(model, 
+        MOI.ScalarAffineFunction(
+            MOI.ScalarAffineTerm.([1.0, 2.0], X), 0.0),
+             MOI.LessThan(3.0))
+    
+    MOI.add_constraint(model, 
+        MOI.ScalarAffineFunction(
+            [MOI.ScalarAffineTerm(1.0, X[1])], 0.0),
+             MOI.GreaterThan(1.0))
+    
+    MOI.add_constraint(model, 
+        MOI.ScalarAffineFunction(
+            [MOI.ScalarAffineTerm(1.0, X[2])], 0.0),
+             MOI.GreaterThan(0.0))
+    
+    MOI.set(model, 
+        MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), 
+        MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([-4.0, -3.0], [X[1], X[2]]), -1.0))
+    
+    MOI.set(model, MOI.ObjectiveSense(), MOI.MIN_SENSE)
+
+    return model
+end    
+
+function lp3_test()
+    #= 
+    min -4x1 -3x2 -1
+      s.a.
```

Comment:
`s.a.`?

Reply:
This is Portuguese for s.t. hahaha "sujeito a". I will make every problem in the following pattern
```julia
    #= 
        objective
    s.t.
        constraints
    =#
```